.idea/mrd-auth0-laravel.iml

```diff
@@ -126,7 +126,6 @@
       <excludeFolder url="file://$MODULE_DIR$/vendor/psr-mock/http-message-implementation" />
       <excludeFolder url="file://$MODULE_DIR$/vendor/psr-mock/http-factory-implementation" />
       <excludeFolder url="file://$MODULE_DIR$/vendor/psr-mock/http-client-implementation" />
-      <excludeFolder url="file://$MODULE_DIR$/vendor/marketredesign/newrelic-monolog-enricher" />
```

Comment:
Seems like ok changes in these .idea files. Where done automatically by PhpStorm.